src/discrete-layer/components/system-status/system-core-info/system-core-info.dialog.css

```diff
@@ -0,0 +1,76 @@
+#systemCoreInfoDialog .mdc-dialog__surface {
+    max-width: unset;
+    right:12px;
+    top:20px;
+    overflow-y: hidden !important;
+    overflow-x: hidden;
+  }
+  
+  #systemCoreInfoDialog .mdc-dialog {
+    justify-content: center;
+  }
+  
+  #systemCoreInfoDialog #systemCoreInfoBody {
+    --tab-bar-height: 50px;
```

Comment:
move it on inline style level
<DialogContent id="systemCoreInfoBody" **style="--tab-bar-height: 50px"**>

Reply:
fixed